Documentation/articles/introduction/platforms.md

```diff
@@ -59,20 +74,46 @@ This platform is meant to publish games on the Windows Store, for both Windows a
 
 Note that UWP games running on Xbox One get [restricted access](https://docs.microsoft.com/en-us/windows/uwp/xbox-apps/system-resource-allocation) to the console capabilities. To unlock those restrictions, MonoGame has a dedicated Xbox One platform for registered [ID@Xbox](https://www.xbox.com/en-US/Developers/id) developers targeting the XDK (this platform is private and requires you to contact your ID@Xbox manager).
 
-## Android
+### Android
 
-**Supported Systems**: Android
-**NuGet Package**: MonoGame.Framework.Android
-**Template ID**: mgandroid
+|**Supported Systems**|**NuGet Package**|**Template ID**|
+|-|-|-|
+|Android | MonoGame.Framework.Android | mgandroid |
 
 The Android platform uses [Xamarin.Android](https://docs.microsoft.com/en-us/xamarin/android/).
 OpenGL is used for graphics, and OpenAL for audio.
 
-## iOS
+### iOS
 
-**Supported Systems**: iOS
-**NuGet Package**: MonoGame.Framework.iOS
-**Template ID**: mgios
+|**Supported Systems**|**NuGet Package**|**Template ID**|
+|-|-|-|
+| iOS | MonoGame.Framework.iOS | mgios |
 
 The iOS platform uses [Xamarin.iOS](https://docs.microsoft.com/en-us/xamarin/ios/).
 OpenGL is used for graphics, and OpenAL for audio.
+
+### NetStandard Library
```

Comment:
I don't think these should go here, since they're not platforms. But it's definitely good to have them documented somewhere.

Reply:
I've added a new header to denote these are additional project types.  As you say, there isn't really anywhere else they could go.